csrc/logical_domain_map.h

```diff
@@ -504,6 +504,10 @@ class ComputeAtLogicalDomainMapBuilder : private BackwardVisitor {
     mapPointwiseLikeOp(op);
   }
 
+  void handle(RepeatOp* op) override {
+    mapPointwiseLikeOp(op);
```

Comment:
Is it supposed to be mapped as pointwise? The input and the output have different extents.